buildutils/src/utils.ts

```diff
@@ -187,7 +187,8 @@ export function checkStatus(cmd: string): number | null {
  */
 export function getPythonVersion(): string {
   const cmd = 'python setup.py --version';
-  return run(cmd, { stdio: 'pipe' }, true);
+  const lines = run(cmd, { stdio: 'pipe' }, true).split('\n');
+  return lines[lines.length - 1];
```

Comment:
Not sure what changed compared to the commit that passed on `master` yesterday with this commit: https://github.com/jupyterlab/jupyterlab/commit/602b05399b0ca762613c8f560a49b15abdefee39

But regardless this should be a better way to handle the output of the command, since we are only interested in the last line. Similar to this change in `jupyter_releaser`: https://github.com/jupyter-server/jupyter_releaser/pull/37